CHANGES.md

```diff
@@ -0,0 +1,23 @@
+# Changes
+
+## v0.0.5
+
+- Verify mailto links. See [#35](https://github.com/Youssef1313/markdown-links-verifier/issues/35).
+- Log invalid links as errors instead of regular text. See [#32](https://github.com/Youssef1313/markdown-links-verifier/issues/32).
```

Comment:
```suggestion
- Log invalid links as errors instead of regular text. See [issue #32](https://github.com/Youssef1313/markdown-links-verifier/issues/32) and [PR #34](https://github.com/Youssef1313/markdown-links-verifier/pull/34).
```